server/app.js

```diff
@@ -91,14 +91,6 @@ app.use(cookieParser(process.env.SESSION_SECRET))
 app.use(bodyParser.json({ limit: '50mb', extended: true }))
 app.use(bodyParser.urlencoded({ limit: '50mb', extended: true }))
 
-/* csrf protection according to http://expressjs.com/en/resources/middleware/csurf.html#simple-express-example */
```

Comment:
Sooooo... we do in fact need CSRF protection, but if this is unused right now that's a separate issue. We should discuss whether we want to continue w/CSRF tokens or switch to an `Authorization: Bearer` header. Either will do the job.

Reply:
This is currently unused right now